src/System.Management.Automation/engine/remoting/client/Job.cs

```diff
@@ -3382,7 +3382,10 @@ internal ErrorRecord FailureErrorRecord
                                 fullyQualifiedErrorId, ErrorCategory.OpenError,
```

Comment:
@PaulHigin, your last commit had 1 failures in `PowerShell-CI-windows`
Enter-PSHostProcess tests.By CustomPipeName.Can enter, exit, and re-enter using CustomPipeName
```powershell
Expected $true, because The script was able to re-enter another process and grab the pipe of 'wrpln4rk.bo1'., but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Enter-PSHostProcess.Tests.ps1: line 187
187:                     Should -BeTrue -Because "The script was able to re-enter another process and grab the pipe of '$pipeName'."
```

Reply:
@PoshChan  Please retry Windows